src/io/utilities/functionSignatures.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
@tpfau , are these `.json` files generated manually? If so, they will be challenging to maintain ...